src/mapping/NearestNeighborGradientMapping.cpp

```diff
@@ -41,12 +43,8 @@ void NearestNeighborGradientMapping::onMappingComputed(mesh::PtrMesh origins, me
     const auto &matchedVertexCoords = searchSpace.get()->vertices()[_vertexIndices[i]].getCoords();
     const auto &sourceVertexCoords  = origins->vertices()[i].getCoords();
 
-    // We calculate the distances uniformly for both mapping constraints as the difference (input - output)
-    if (hasConstraint(CONSERVATIVE)) {
-      _offsetsMatched[i] = sourceVertexCoords - matchedVertexCoords;
-    } else {
-      _offsetsMatched[i] = matchedVertexCoords - sourceVertexCoords;
-    }
+    // We calculate the distances uniformly for consistent mapping constraint as the difference (output - input)
+    _offsetsMatched[i] = sourceVertexCoords - matchedVertexCoords;
```

Comment:
The comment claims output - input, but you write 'source - matched', is this correct?

Reply:
You could also write your comment more explicit in this regard so that its clear to everyone.